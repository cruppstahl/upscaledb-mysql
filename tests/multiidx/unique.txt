DROP TABLE IF EXISTS test;
CREATE TABLE test (value INTEGER NOT NULL, UNIQUE INDEX(value), num INTEGER PRIMARY KEY) ENGINE=upscaledb;
INSERT INTO test VALUES(1, 1);
INSERT INTO test VALUES(2, 2);
INSERT INTO test VALUES(3, 3);
INSERT INTO test VALUES(3, 4);
INSERT INTO test VALUES(3, 5);
INSERT INTO test VALUES(4, 6);
SELECT * FROM test;
SELECT * FROM test WHERE num >= 3;
UPDATE test SET value = 1 WHERE num = 2;
SELECT * FROM test;
