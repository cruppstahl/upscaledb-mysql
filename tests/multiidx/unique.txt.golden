--------------
DROP TABLE IF EXISTS test
--------------

--------------
CREATE TABLE test (value INTEGER NOT NULL, UNIQUE INDEX(value), num INTEGER PRIMARY KEY) ENGINE=upscaledb
--------------

--------------
INSERT INTO test VALUES(1, 1)
--------------

--------------
INSERT INTO test VALUES(2, 2)
--------------

--------------
INSERT INTO test VALUES(3, 3)
--------------

--------------
INSERT INTO test VALUES(3, 4)
--------------

--------------
INSERT INTO test VALUES(3, 5)
--------------

--------------
INSERT INTO test VALUES(4, 6)
--------------

--------------
SELECT * FROM test
--------------

value	num
1	1
2	2
3	3
4	6
--------------
SELECT * FROM test WHERE num >= 3
--------------

value	num
3	3
4	6
--------------
UPDATE test SET value = 1 WHERE num = 2
--------------

--------------
SELECT * FROM test
--------------

value	num
1	1
2	2
3	3
4	6
